File: WebApplication/src/modules/student/dtos/person-lot.dtos/person-secondary.dtos/person-secondary.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";

export class PersonSecondaryDto {
  graduationYear: number;
  country: NomenclatureDto;
  school: NomenclatureDto;
  schoolSettlement: NomenclatureDto;
  foreignSchoolName: string;
  profession: string;
  diplomaNumber: string;
  diplomaDate: Date;
}

File: StudentCard.Application/Students/Dtos/PersonLotDtos/PersonSecondaryDtos/PersonSecondaryDto.cs
using StudentCard.Infrastructure.Helpers.Dtos;
using System;

namespace StudentCard.Application.Students.Dtos.PersonLotDtos.PersonSecondaryDtos
{
    public class PersonSecondaryDto
    {
        public int GraduationYear { get; set; }
        public NomenclatureDto Country { get; set; }
        public NomenclatureDto School { get; set; }
        public NomenclatureDto SchoolSettlement { get; set; }
        public string ForeignSchoolName { get; set; }
        public string Profession { get; set; }
        public string DiplomaNumber { get; set; }
        public DateTime? DiplomaDate { get; set; }
    }
}
